doc/1.4/language.md

```diff
@@ -987,22 +987,42 @@ Templates are imported into an object declaration body using
 <pre>
 is <em>name</em>;
 </pre>
+for example:
+```
+field F {
+    is A;
+}
+```
 
 It is also possible to use templates when declaring an object, as in
 
 <pre>
 field F is (<em>name1</em>, <em>name2</em>);
 </pre>
 
-These can be used in any context where an object declaration may be
-written, and has the effect of expanding the body of the template at
-the point of the `is`. If two templates define methods or
-parameters with the same name, then the template instantiation
-hierarchy is used to deduce which method overrides the other: If one
-template *B* instantiates another template *A*, directly or indirectly,
-then methods from *B* override methods from *A*. Note, however, that
-overrides can only happen on methods and parameters that are
-declared `default`. Example:
+These can be used in any context where an object declaration may be written, and
+has the effect of expanding the body of the template at the point of the `is`.
+Using `is` together with object declarations is typically more idiomatic than
+the standalone `is` object statement, but the latter is very useful in
+conjunction with [`in each` declarations](#in-each-declarations); for example:
```

Comment:
Another use case is top level, where standalone `is` is the only way to instantiate templates.

Reply:
Also, the `in each { is }` example is a bit weird; we had a discussion on unifying syntaxes for object-like declarations to allow `in each field is A;`, `template a is b;`; the conclusion was that this would not be wrong in principle, but the use cases are so far too weak (together with irregularities such as `desc` string shorthand only making sense on concrete object declarations).

If we think that the in each use case is a too weak use case for adding an `in each field is A { ... }`, then it is inconsistent to present it as the primary motivation for `in each field { is A; }`; we could just as well add the former form (unless we think the latter looks better).